### DISCUSSION TITLE
Styling Issue using Tailwind and Vite

### DISCUSSION THREAD

**Discussion by omar-qayum** [wojtekmaj/react-date-picker#562] [General]
Hi!

I read the docs to the best of my abilities but I am still having issues.

When I import the DatePicker from 'react-date-picker/dist/entry.nostyle', I still get all the default styles. The npm page says: 
**"Next.js and Vite do not allow components from node_modules to import styles. You'll need to follow instructions from [Custom styling](https://www.npmjs.com/package/react-date-picker#custom-styling) to get going."**

That is what I am following. Did I get something wrong? 

Basically, I want to remove the default border on the date picker which is being provided by the .react-date-picker__wrapper class. I am using Tailwind, but any style I apply to className doesn't override the CSS properties of that class.

Thanks!

**Comment by omar-qayum** [comment]
Nevermind, I simply overrode the CSS styles with my own :)

Thx for the hard work!
